Tasks/DownloadPipelineArtifactV2/task.json

```diff
@@ -0,0 +1,229 @@
+{
+    "id": "61F2A582-95AE-4948-B34D-A1B3C4F6A737",
+    "name": "DownloadPipelineArtifact",
+    "friendlyName": "Download pipeline artifact",
+    "description": "Download build and pipeline artifacts",
+    "helpMarkDown": "Download one or more published build or pipeline artifacts.",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/download-pipeline-artifact",
+    "category": "Utility",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 2,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "groups": [],
+    "demands": [],
+    "preview": true,
+    "minimumAgentVersion": "2.153.1",
+    "inputs": [
+        {
+            "name": "source",
+            "aliases": [
+                "buildType"
+            ],
+            "type": "radio",
+            "label": "Download artifacts produced by",
+            "defaultValue": "current",
+            "required": true,
+            "options": {
+                "current": "Current run",
+                "specific": "Specific build"
+            },
+            "helpMarkDown": "Download artifacts produced by the current build, or from a specific build."
```

Comment:
>Download artifacts produced by the current build, or from a specific build. [](start = 29, length = 75)

Download artifacts produced by the current pipeline run, or from a specific pipeline run.